src/main/groovy/wooga/gradle/version/internal/DefaultVersionPluginExtension.groovy

```diff
@@ -284,7 +284,7 @@ class DefaultVersionPluginExtension implements VersionPluginExtension {
                             VersionConsts.DEFAULT_MAIN_BRANCH_PATTERN).toString()
         }))
 
-        version = new MemoisationProvider(project.provider({
+        version = new MemoizationProvider(project.provider({
```

Comment:
That whole thing is actually not needed. I figured out waaaay to late that groovy closure as `memoize` method.